draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -44,10 +44,16 @@
   private static final float SLIDE_TOP = 0f;
   private static final float SLIDE_BOTTOM = 1f;
   private static final boolean DEFAULT_ENABLE_HORIZONTAL_ALPHA_EFFECT = true;
+  private static final boolean DEFAULT_ENABLE_CLICK_TO_MAXIMIZE = false;
+  private static final boolean DEFAULT_ENABLE_CLICK_TO_MINIMIZE = false;
+  private static final int MIN_SLIDING_DISTANCE_ON_CLICK = 10;
   private static final int ONE_HUNDRED = 100;
   private static final float SENSITIVITY = 1f;
   private static final boolean DEFAULT_TOP_VIEW_RESIZE = false;
 
+  private float lastTouchActionDownXPosition;
+  private int minSlidingDistanceOnClick = 10;
```

Comment:
If you are going to create a DEFAULT_SLIDING_DISTANCE_ON_CLICK constant, initialize this attribute with that value.